mailbox/store/src/test/java/org/apache/james/mailbox/store/search/AbstractMessageSearchIndexTest.java

```diff
@@ -1099,9 +1099,9 @@ public void sortOnToShouldWork() throws Exception {
         assertThat(messageSearchIndex.search(session, mailbox, searchQuery))
             .containsExactly(m5.getUid(), m3.getUid(), m2.getUid(), m4.getUid());
         // 5 : "zzz" <mailet-api@james.apache.org>
-        // 3 : "aaa" <server-dev@james.apache.org>
-        // 2 : "abc" <server-dev@james.apache.org>
-        // 4 : "server" <server-dev@james.apache.org>
+        // 3 : "aaa" <a-server-dev@james.apache.org>
```

Comment:
could you elaborate? Why do we change our expectations?

Reply:
I think that we wanted to show that the display name was used for the ordering

Reply:
Well...

SortConverter enforce only `Adress sorting ` upon MailboxTo.

Note that we also have `DisplayTo` for sorting on display.

So the behavior you describe, that happened to maybe be true accidentaly, can also be reproduced by soting on MailboxTo then on DisplayTo.

Not an issue to me.